src/config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "GUILD": {
+        "ID": 742797665301168220,
+        "ROLES": {
+            "admin": 742800061280550923,
+            "mod": 742798158966292640,
+            "botDev": 750556082371559485,
+            "verified": 749683320941445250,
+            "just_joined": 798765678739062804,
+            "muted": 775981947079491614
+        },
+        "CHANNELS": {
+            "LOGS": {
+                "logs": 786084620944146504,
+                "modlogs": 778678059879890944,
+                "nqnlogs": 927077979383824484
+            },
+            "welcomeChannel": {
+                "ID": 742946580285620225
+            },
+            "lobbyChannel": {
+                "ID": 860224115633160203
+            }
+        }
+    }
+}
```

Comment:
Enhancement: every key name must be all caps here. Ideally, it should follow [this](https://github.com/sach-12/pesu-bot-verify/blob/app-router/src/utils/config.js#L6) exact format for uniformity